libr/core/cmd.c

```diff
@@ -5953,6 +5983,356 @@ DEFINE_HANDLE_TS_FCN(iter_hit_command) {
 	return res;
 }
 
+DEFINE_HANDLE_TS_FCN(foreach_addrsize_command) {
+	RCore *core = state->core;
+	TSNode command = ts_node_named_child (node, 0);
+	uint32_t i = 1;
```

Comment:
use size_t

Reply:
tree-sitter uses `uint32_t` so I thought I should use it as well to avoid problems. `size_t` may be ut64 on some arch, while uint32_t is always 32bit.